### ISSUE TITLE
numpy version is incompatible

### ISSUE THREAD

**Issue by leon0707** [RasaHQ/rasa_core#1023]
**Rasa Core version**:
0.11.4
**Python version**: 
3.5.6
**Operating system** (windows, osx, ...):
osx
**Issue**:
```
tensorflow 1.10.0 has requirement numpy<=1.14.5,>=1.13.3, but you'll have numpy 1.15.1 which is incompatible.
```
rasa_core requirements.txt
```
numpy==1.15.1
```
Version of numpy is either incompatible with tensorflow or rasa_core

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by marekkosta** [comment]
The same issue here:

rasa-nlu: 0.13.4
rasa-core: 0.11.4
python: 3.6.6
OS: Ubuntu 16.04.5 LTS

Through Pipfile with the following content:
```
[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true
name = "pypi"

[packages]
rasa-nlu = "==0.13.4"
rasa-core = "==0.11.4"

[dev-packages]

[requires]
python_version = "3.6"
```
Running ``pipenv install`` runs into the same conflicting conditions as mentioned by @leon0707 .
The result is that the ``pipenv install`` command does not install anything at all...
Its output is:
```
$ pipenv install
Pipfile.lock not found, creating...
Locking [dev-packages] dependencies...
Locking [packages] dependencies...

Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
Could not find a version that matches numpy<=1.14.5,==1.15.1,>=1.12.0,>=1.13,>=1.13.3,>=1.7,>=1.7.1,>=1.8.2,>=1.9.1,~=1.15
Tried: 1.3.0, 1.4.1, 1.5.0, 1.5.1, 1.5.1, 1.6.0, 1.6.0, 1.6.0, 1.6.0, 1.6.0, 1.6.0, 1.6.0, 1.6.1, 1.6.1, 1.6.1, 1.6.1, 1.6.1, 1.6.1, 1.6.1, 1.6.2, 1.6.2, 1.6.2, 1.6.2, 1.6.2, 1.6.2, 1.6.2, 1.7.0, 1.7.0, 1.7.0, 1.7.0, 1.7.0, 1.7.0, 1.7.0, 1.7.0, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.1, 1.7.2, 1.7.2, 1.7.2, 1.7.2, 1.7.2, 1.7.2, 1.7.2, 1.7.2, 1.8.0, 1.8.0, 1.8.0, 1.8.0, 1.8.0, 1.8.0, 1.8.0, 1.8.0, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.1, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.8.2, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.0, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.1, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.2, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.9.3, 1.10.0, 1.10.0, 1.10.0, 1.10.0, 1.10.0, 1.10.0, 1.10.0, 1.10.0.post2, 1.10.0.post2, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.1, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.2, 1.10.3, 1.10.3, 1.10.3, 1.10.3, 1.10.3, 1.10.3, 1.10.3, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.10.4, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.0, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.1, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.2, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.11.3, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.0, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.12.1, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.0, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.1, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.13.3, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.0, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.1, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.2, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.3, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.4, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.14.5, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.0, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1, 1.15.1
Skipped pre-versions: 1.11.0b3, 1.11.0b3, 1.11.0b3, 1.11.0b3, 1.11.0b3, 1.11.0b3, 1.11.0rc1, 1.11.0rc1, 1.11.0rc1, 1.11.0rc1, 1.11.0rc1, 1.11.0rc1, 1.11.0rc2, 1.11.0rc2, 1.11.0rc2, 1.11.0rc2, 1.11.0rc2, 1.11.0rc2, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.1rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.11.2rc1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0b1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc1, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.0rc2, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.12.1rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc1, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.13.0rc2, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.14.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc1, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2, 1.15.0rc2
There are incompatible versions in the resolved dependencies.
```
This contrasts with the behavior of the rasa-core recommended installation approach ``pip install -r requirements.txt``, which emits the warning, but at the end of the day ``pip list`` reports that
```
numpy               1.15.1
```
is installed.

**Comment by tmbo** [comment]
Yes that is a very valid point, tensorflow already added a fix on their end, will ship with 1.11 (there is already a rc for that so I guess in about a week). I am not sure if we actually require numpy 1.15 - best guess is no, so might make sense to create a PR for that :+1:

**Comment by tmbo** [comment]
Nevermind, I just noticed how this got introduced and created a fix. version 0.11.5 is currently building and will fix this issue (numpy reverted to 0.14.5).
